static/styles/subscriptions.css

```diff
@@ -1069,6 +1065,15 @@ h4.stream_setting_subsection_title {
     }
 }
 
+.stream-creation-body input[type="radio"] {
+    margin: 3.5px 0 1px;
+}
+
+/* we have this 2px difference due to the font size being larger in the modal */
+#stream_privacy_modal input[type="radio"] {
+    margin: 5.5px 0 1px;
+}
```

Comment:
This is OK as a tactical fix, but ideally we'd just have our radio components declaratively vertically center the radio input rather than needing to specify the correct number of pixels here.

Reply:
(But I'm not sure we need to go down that rabbit hole; we only have 3 radio inputs in the whole product -- these two plus the emoji set one).